### DISCUSSION TITLE
Add Shift accelerator to Arrow key movement of Control Points

### DISCUSSION THREAD

**Discussion by Q210108** [opentoonz/opentoonz#6834] [Ideas]
that s a little suggestion we sometimes need

**Comment by RodneyBaker** [comment]
I note that we do have the Shift constraint for movement of column objects vertically and horizontally but we do not have that same functinality for movement of control points via the Control Point Editor.  As such the basic code for this feature request may already be in place.

@pojienie You may have some insight into this as you've spent considerable time in that area.

**Comment by artisteacher** [comment]
For now we can use the arrow keys for horizontal/vertical movement of the control points. Though adding a shift modifier to the arrow keys to speed up the horizontal/vertical movement would be both a standard behavior & helpful.
